packages/app/src/vis-packs/nexus/pack-utils.ts

```diff
@@ -2,57 +2,36 @@ import {
   assertGroupWithChildren,
   assertStr,
   buildEntityPath,
-  handleError,
-  hasChildren,
   hasComplexType,
   hasMinDims,
   hasNumDims,
   isGroup,
 } from '@h5web/shared';
 import type { Entity } from '@h5web/shared';
-import type { FetchStore } from 'react-suspense-fetch';
 
-import { ProviderError } from '../../providers/models';
 import { getAttributeValue } from '../../utils';
 import type { VisDef } from '../models';
 import { NxInterpretation } from './models';
 import { findSignalDataset, isNxDataGroup, isNxGroup } from './utils';
 import { NexusVis, NEXUS_VIS } from './visualizations';
 
-function getDefaultEntityPath(entity: Entity): string | undefined {
```

Comment:
Basically `getDefaultEntity` becomes `resolvePath` but with the visualisation support check added, and `getDefaultEntityPath` becomes `getNxDefaultPath`.